src/elements/mixin/pipeaperture.H

```diff
@@ -60,42 +62,43 @@ namespace impactx::elements::mixin
             using namespace amrex::literals; // for _rt and _prt
 
             // skip aperture application if aperture_x <= 0 or aperture_y <= 0
-            if (m_aperture_x > 0 && m_aperture_y > 0) {
+            if (m_inv_aperture_x > 0_prt && m_inv_aperture_y > 0_prt) {
 
                // scale horizontal and vertical coordinates
```

Comment:
```suggestion
               // scale horizontal and vertical coordinates
               // performance optimization: we intentionally store the inverse of
               //   the aperture, so we can do a quick multiplication (instead of a
               //   costly division) here.
```